src/main/java/com/cegeka/academy/web/rest/ChallengeController.java

```diff
@@ -57,6 +57,13 @@ public void saveChallenge(@Valid @RequestBody ChallengeDTO challenge) {
 
     }
 
+    @GetMapping("/next")
+    public List<ChallengeDTO> getNextChallenges() throws NotFoundException {
+
+        return challengeService.getNextChallenges();
```

Comment:
Dar nu trebuie sa afisam doar acele challenge-uri care sunt in viitor pentru un user anume? Daca sunt private si sunt in viitor dar el nu are invitatie nu ar trebui sa le vada. Ai acoperit acest caz?